src/status_im/protocol/handlers.cljs

```diff
@@ -495,13 +495,12 @@
          {:keys [group-id identity message-id timestamp]} :payload}]]
     (let [admin (get-in chats [group-id :group-admin])]
       (when (= from admin)
-        (merge
-          {::participant-invited-to-group-message {:group-id group-id :current-public-key current-public-key
-                                                   :identity identity :from from :message-id message-id
-                                                   :timestamp timestamp}}
-          (when-not (and (= current-public-key identity) has-contact?)
-            {:db (update-in db [:chats group-id :contacts] conj {:identity identity})
-             ::chats-add-contact [group-id [identity]]}))))))
+        (merge {::participant-invited-to-group-message {:group-id group-id :current-public-key current-public-key
```

Comment:
This is a bit hard to read, has anything materially changed here? If no, can we leave it alone? If yes, can we maybe put some of this stuff in lets / use threading etc?

Reply:
the last statement: identity was in a vector before